tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -266,7 +265,11 @@ class MklCPUAllocator : public Allocator {
 
  private:
   // Hooks provided by this allocator for memory allocation routines from MKL
-
+  bool always_use_system_allocator_ = [] {
+    bool value = false;
+    TF_CHECK_OK(ReadBoolFromEnvVar("TF_USE_SYSTEM_ALLOCATOR", false, &value));
```

Comment:
The name should probably be "TF_ONEDNN_USE_SYSTEM_ALLOCATOR" (or "TF_MKL_USE_SYSTEM_ALLOCATOR", etc.) since this only controls TF-oneDNN code.